src/components/views/elements/PollCreateDialog.tsx

```diff
@@ -89,7 +89,8 @@ export default class PollCreateDialog extends ScrollableBaseModal<IProps, IState
         this.setState({ options: newOptions }, () => {
             // Scroll the button into view after the state update to ensure we don't experience
             // a pop-in effect, and to avoid the button getting cut off due to a mid-scroll render.
-            this.addOptionRef.current?.scrollIntoView();
+            this.addOptionRef.current?.scrollIntoView &&
+                this.addOptionRef.current?.scrollIntoView();
```

Comment:
```suggestion
            this.addOptionRef.current?.scrollIntoView?.();
```

Reply:
Ah brilliant, thank you - I tried to do than and bungled it by missing the dot.